modules/katamari/src/main/ts/ephox/katamari/api/Regex.ts

```diff
@@ -0,0 +1 @@
+export const escapeRegExp = (text: string): string => text.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```

Comment:
Since we're making this, should we move over other other regex'es while we're at it? The original issue seems to be wrong, we don't use escapeRegExp in `SmartPaste`, but we do use another regex there, and I'm sure there's a few other 'standard' ones around. Or do we want to handle that in other issues as we go along?

Reply:
A lot of the other 'standard' regexes are in Polaris, which is designed to be "the package that has regexes / parsers in it." I think the benefit of a separate katamari regex module is that we can use it to gather more meta-functions. In other words, I think this is a good place to put functions that **operate on** regexes, but I don't think this is a good place to put functions that are **implemented using** regexes (except in cases like this where the fact that it's implemented using a regex is a coincidence)